tinyiiod-private.h

```diff
@@ -21,30 +21,30 @@
 #include "tinyiiod.h"
 
 char tinyiiod_read_char(struct tinyiiod *iiod);
-void tinyiiod_read(struct tinyiiod *iiod, char *buf, size_t len);
-ssize_t tinyiiod_read_line(struct tinyiiod *iiod, char *buf, size_t len);
+int32_t tinyiiod_read_line(struct tinyiiod *iiod, char *buf);
 
 void tinyiiod_write_char(struct tinyiiod *iiod, char c);
 void tinyiiod_write(struct tinyiiod *iiod, const char *data, size_t len);
 void tinyiiod_write_string(struct tinyiiod *iiod, const char *str);
-void tinyiiod_write_value(struct tinyiiod *iiod, int value);
+void tinyiiod_write_value(struct tinyiiod *iiod, int32_t value);
```

Comment:
`int` is a more standard type that `int32_t` ; so, i would not change this ;
if we want to change this, we may need a `compat.h` layer to make sure that these types (`int32_t` & friends) exist; 

i agree that this code already has `ssize_t` types, which i'm also not sure how standard they are ;

in any case, if we want to fix/address int/int32_t/size_t types, we should do it in separate commits, and focus the discussion only on these